src/index.css

```
@@ -259,23 +269,32 @@ body {
     @apply fixed left-4 top-2 z-[60] -translate-y-[300%] rounded-md border border-border-base bg-bg-surface px-4 py-2 font-semibold text-primary shadow-glass transition-transform duration-200 focus-visible:translate-y-0;
   }
 
+  /* The transparent border is not decoration: btn-quiet carries a real one, and
+     without a matching border the filled buttons stand 2px shorter in the same
+     stack. */
   .btn-primary {
-    @apply bg-primary text-text-on-primary shadow-md hover:bg-primary-hover disabled:opacity-60 disabled:cursor-not-allowed disabled:shadow-none;
+    @apply bg-primary text-text-on-primary border border-transparent shadow-md hover:bg-primary-hover disabled:opacity-60 disabled:cursor-not-allowed disabled:shadow-none;
     box-shadow: 0 4px 6px -1px hsla(var(--hue-primary), 60%, 40%, 0.2);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)